experiments/long_runs/low_res_snowy_land_rh.jl

```diff
@@ -65,7 +65,6 @@ function setup_model(FT, start_date, stop_date, Δt, domain, toml_dict)
         -0.2,
         FT;
         max_wind_speed = 25.0,
-        time_interpolation_method = LinearInterpolation(PeriodicCalendar()),
```

Comment:
`prescribed_perturbed_rh_era5` and `prescribed_perturbed_temperature_era5` still use this as their default - should they be updated to use `LinearInterpolation(PeriodicCalendar(Dates.Year(1), DateTime(Dates.year(stop_date))))` too?